truss/templates/control/control/helpers/inference_server_controller.py

```diff
@@ -63,8 +64,25 @@ def apply_patch(self, patch_request):
                 Patch.from_dict(patch_dict) for patch_dict in patch_request["patches"]
             ]
             self._process_controller.stop()
-            for patch in patches:
-                self._patch_applier.apply_patch(patch)
+            try:
+                patches_executed = 0
+                for patch in patches:
+                    self._patch_applier.apply_patch(patch)
```

Comment:
should we try to catch exceptions on a per patch basis, vs on the batch of patches? I'm not I understand what the intention is here

Reply:
Partially applied patches are actually problematic, we don't some patches to apply vs others, want to exit at the earliest.